app/services/api/v3/delivery_partners/show.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Api
+  module V3
+    module DeliveryPartners
+      class Show
+        attr_reader :lead_provider, :params
+
+        def initialize(lead_provider:, params:)
+          @lead_provider = lead_provider
+          @params = params
+        end
+
+        def delivery_partner
+          lead_provider.delivery_partners.find_by("delivery_partners.id = ?", params[:id])
+        end
+
+      private
+
+        def filter
```

Comment:
I don't think we need filters/with cohorts here?
So regarding where we put this, let's discuss tomorrow as in the PR Im working on I included it in the existing query service, as keen to get your thoughts

Reply:
Yeah I forgot to delete.. all sorted now.. tks @cwrw